src/ndn/transport/face.py

```diff
@@ -18,8 +18,9 @@
 import io
 import abc
 import asyncio as aio
+import logging
```

Comment:
Sort imports with `isort`.
Moreover, you should add autopep8 and isort into CI scripts to prevent future mistakes.